docs/installation.md

```diff
@@ -143,7 +143,7 @@ Then, build the containers locally.
 From the root folder of the repo:
 
 ```sh
-docker-compose --build
+docker-compose up --build

```

Comment:
The command after (in line 152) does the `up` part, but you're right there's a problem here. 
I would suggest to join the two and just call `docker-compose up --build`